dts/msm8916/msm8939-xiaomi-ido.dts

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/dts-v1/;
+
+#include <skeleton.dtsi>
+
+/ {
+	qcom,msm-id = <0xef 0x30000 0xf1 0x30000 0x107 0x30000>;
+	qcom,board-id = <0x1000b 11>, <0x2000b 11>;
+
+	model = "Xiaomi Redmi 3";	
+	compatible = "xiaomi,ido", "qcom,msm8939", "lk2nd,device";
+
+
+	panel {
+		qcom,mdss_dsi_ili9881c_tianma_720p_video {
```

Comment:
Please assign `compatible`s to each supported panel, similar to `msm8916-qrd7+12-v1.dts`